rules/windows/defense_evasion_execution_suspicious_explorer_winword.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an instance of Explorer or Winword starting after being renamed or from a non-standard path. This is uncommon
+behavior and may indicate an attempt to evade defenses via side loading a malicious DLL within the memory
+space of one of those processes. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential DLL SideLoading via Trusted Microsoft Programs"
+risk_score = 73
+rule_id = "1160dcdb-0a0a-4a79-91d8-9b84616edebd"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ process.pe.original_file_name:(WinWord.exe or EXPLORER.EXE) and
```

Comment:
any reason to not include more office programs?

Reply:
only apply to winword.exe with wwlib.dll